packages/documentation/src/stories/foundation/layout/Columns/columns.stories.ts

```diff
@@ -0,0 +1,252 @@
+        'justify-content-around',
+        'justify-content-between',
+        'justify-content-evenly',
+      ],
+      table: {
+        category: 'General',
+      },
+    },
+    offsetItem: {
+      name: 'Offset classes',
+      description: 'Increases the left margin of a column',
+      control: {
+        type: 'select',
+      },
+      options: [
+        'offset-1',
+        'offset-2',
+        'offset-3',
+        'offset-4',
+        'offset-5',
+        'offset-6',
+        'offset-7',
+        'offset-8',
+        'offset-9',
+        'offset-10',
+        'offset-11',
+      ],
+      table: {
+        category: 'General',
+      },
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+export const VerticalExample: Story = {
+  parameters: {
+    controls: { exclude: ['Horizontal Alignement', 'Offset classes'] },
+  },
+  render: (args: Args) => html`
+    <div class="container">
+      <div class="row-height row ${args.alignItems}">
+        <div class="col${args.alignSelf === 'no self alignment' ? '' : ` ${args.alignSelf}`}">
+          Item 1
+        </div>
+        <div class="col">Item 2</div>
+        <div class="col">Item 3</div>
+      </div>
+    </div>
+  `,
+};
+
+export const HorizontalExample: Story = {
+  parameters: {
+    controls: { exclude: ['Align Items', 'Align Item 1', 'Offset classes'] },
+  },
+  render: (args: Args) => html`
+    <div class="container">
+      <div class="row ${args.justifyContent}">
+        <div class="col-4">Item 1</div>
+        <div class="col-4">Item 2</div>
+      </div>
+    </div>
+  `,
+};
+
+export const OrderExample: Story = {
+  render: () => html`
+    <div class="container">
+      <div class="row">
+        <div class="col">First in DOM, no order applied</div>
+        <div class="col order-5">Second in DOM, with a larger order</div>
+        <div class="col order-1">Third in DOM, with an order of 1</div>
+      </div>
+    </div>
+  `,
+};
+
+export const OrderMaxExample: Story = {
+  render: () => html`
+    <div class="container">
+      <div class="row">
+        <div class="col order-last">First in DOM, ordered last</div>
+        <div class="col">Second in DOM, unordered</div>
+        <div class="col order-first">Third in DOM, ordered first</div>
+      </div>
+    </div>
+  `,
+};
+
+export const OffsetExample: Story = {
+  parameters: {
+    controls: { exclude: ['Align Items', 'Align Item 1', 'Horizontal Alignement'] },
+  },
+  render: (args: Args) => html`
+    <div class="row">
+      <div class="col-1 ${args.offsetItem}">.col-1 .${args.offsetItem}</div>
+    </div>
+  `,
+};
+
+export const ColumnBreakExample: Story = {
+  render: () => html`
+    <div class="row">
+      <div class="col-3">.col-3</div>
+      <div class="col-3">.col-3</div>
+
+      <!-- Force next columns to break to new line -->
+      <div class="w-100"></div>
+
+      <div class="col-3">.col-3</div>
+      <div class="col-3">.col-3</div>
+    </div>
+  `,
+};
```

Comment:
Can we add controls to show how it would look, without the `div.w-100` element, this would be great!
For example:

```ts
export const ColumnBreakExample: Story = {
  args: {
    renderBreakingElement: true
  },
  argTypes: {
    renderBreakingElement: {
      name: 'Render breaking element',
      description: 'Toggle rendering of breaking element',
      control: {
        type: 'boolean',
      }
    }
  },
  render: (args: Args) => html`
    <div class="row">
      <div class="col-3">.col-3</div>
      <div class="col-3">.col-3</div>
      <!-- Force next columns to break to new line -->
      ${args.renderBreakingElement ? html`<div class="w-100"></div>` : nothing}
      <div class="col-3">.col-3</div>
      <div class="col-3">.col-3</div>
    </div>
  `,
};
```